# 08 Inventory tails

Do the outer deciles of the WCESTUS1 surprise, cut on the train window only, still separate five-print RWTC returns after 2019?

Holdout starts 2020-01-01. Verdict: held.

This study reads stored series. It does not add a derived column.

In [1]:
import json
from pathlib import Path
studies = json.loads(Path('data/studies.json').read_text())
study = next(item for item in studies['studies'] if item['slug'] == 'events')
print(study['verdict'])
print(study['verdict_line'])
print(study['table']['caption'])
for row in study['table']['rows']:
    print(' | '.join(row))


held
Frozen train tails are -5648 and 5678 thousand barrels. Train draw-minus-build five-print return is -0.348%. Holdout difference is -2.035%. A negative number means the draw tail was followed by a lower return than the build tail. The holdout kept the train sign.
Five-print RWTC return by frozen WCESTUS1 surprise tail
Train | draw | 159 | -0.169%
Train | middle | 1266 | 0.181%
Train | build | 159 | 0.179%
Holdout | draw | 47 | -1.377%
Holdout | middle | 255 | 0.319%
Holdout | build | 45 | 0.658%


## Method

The 10th and 90th percentiles are nearest ranks of the train surprises. Holdout weeks are labeled with those two numbers.

## Result

Frozen train tails are -5648 and 5678 thousand barrels. Train draw-minus-build five-print return is -0.348%. Holdout difference is -2.035%. A negative number means the draw tail was followed by a lower return than the build tail. The holdout kept the train sign.